src/scripts/components/Drawer.js

```diff
@@ -33,8 +58,13 @@ export const Drawer = ( { focus, blur, instance } ) => {
 			<Resizable
 				size={ { width, height: '100%' } }
 				enable={ { left: true } }
-				onResizeStop={ ( e, direction, ref, d ) => {
-					setWidth( width + d.width );
+				onResizeStop={ ( _e, _direction, _ref, d ) => {
+					const currentWidth = width + d.width;
+					if ( currentWidth > window.innerWidth ) {
+						setWidth( window.innerWidth );
+					} else {
+						setWidth( width + d.width );
+					}
 				} }
 				minWidth={ HUB_WIDTH }
```

Comment:
since the drawer during resize could be larger than the viewport could you kindly add this (seems to fix the issue)

![resize](https://user-images.githubusercontent.com/8550908/232313579-24db6d0c-20b3-4ff5-b062-35af04bb0f93.gif)

```js
maxWidth={ '100vw' }
```

Reply:
furthermore ( can be seen in the gif above) we need to set something (like below) because the padding seems to be added to the width of the drawer

```css
#wp-notifications-hub {
  box-sizing: border-box;
}
```

Reply:
scratch this ☝️... maybe it needs a drawer css style rework and is better to do it in a different pr

Reply:
It looks better on my computer, so it must be a browser specific thing. I'm using Arc (which is Chromium)

Reply:
Mine is Canary Chrome (3 versions ahead of Chrome), maybe too far ahead! 😅